permissions.py

```diff
@@ -39,5 +39,5 @@ def has_permission(self, request, view):
         if request.auth is None:
             return False
         token = Token.objects.get(key=request.auth.key)
-        is_noob = token.user.user_type == 'noob'
+        is_noob = token.user_data.user_type == 'noob'
```

Comment:
Kindly change this back to:

`is_noob = token.user.user_type == "noob"`

and do the same for the rest of the permissions written.